app/helpers/emails_helper.rb

```diff
@@ -31,8 +31,8 @@ def action_title(url)
   end
 
   def color_email_diff(diffcontent)
-    formatter = Rugments::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
-    lexer = Rugments::Lexers::Diff.new
+    formatter = Rouge::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
```

Comment:
ugh, yes it's probably a packaging issue.  is it causing problems for it to be there?